libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryUtils.java

```diff
@@ -61,4 +64,30 @@ private static String obtainApplicationIdentifier(Context context) {
       return EMPTY_STRING;
     }
   }
+
+  private static SharedPreferences obtainSharedPreferences() {
```

Comment:
NIT We normally organize the code in this particular order :point_right: `public`, `protected`, `private`, etc. trying to locate them in order of calling. Could we restructure the methods of `TelemetryUtils` this fashion?